content/docs/started/workstation/minikube-linux.md

```diff
@@ -301,4 +303,27 @@ Read the following before execution of the notebook.
         ```
         hostPath:
           path: /mnt/data
-        ```
\ No newline at end of file
+        ```
+
+## Hints and Tips
+### Access a Jupyter Notebook GUI remotely
+If your Jupyter Notebook server is hosted in a Linux machine and you want to access a notebook GUI from your Windows/Mac, you can use the following port forwarding technique.
+
+   General syntax for port forwarding from XXXX to YYYY
+   ```
+   ssh -N -f -L localhost:YYYY:localhost:XXXX remoteuser@remotehost
+   ```
+
+Below is an example of accessing a Jupyter notebook hosted in a Linux machine remotehost.com (port 8888) in a Window machine using port 8889. 
```

Comment:
Change "Window" to "Windows", here and below.

Reply:
Thanks again. Will change.